pkg/util/kubernetes/apiserver/services.go

```diff
@@ -152,9 +152,9 @@ func (metaBundle *MetadataMapperBundle) ServicesForPod(ns, podName string) ([]st
 	metaBundle.m.RLock()
 	defer metaBundle.m.RUnlock()
 
-	svcs, err := metaBundle.Services.Get(ns, podName)
-	if err != nil {
-		log.Errorf("could not get services: %s", err)
+	svcs, ok := metaBundle.Services.Get(ns, podName)
+	if !ok {
+		log.Debugf("could not get services for pod %s in namespace %s", podName, ns)
 		return nil, false
 	}
 	return svcs, true
```

Comment:
isn't it `return metaBundle.Services.Get(ns, podName)` directly?

Reply:
Yeah that is probably an overkill log I have in there.